traiderdaive/ray_environments/ray_hyperdrive_env.py

```diff
@@ -112,21 +125,21 @@ def __init__(
         """Initializes the environment"""
         self.worker_index = env_config.worker_index
```

Comment:
Does this worker index get set within ray? Are there multiple workers making multiple copies of this environment running in parallel?

Reply:
Yes, this class gets wrapped by ray, and init gets called with env_config which has a worker index. There is a copy of this environment for each worker.